<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNovaMaquina</class>
 <widget class="QDialog" name="DialogNovaMaquina">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget_Paisos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_50">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>50 Hz</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_60">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>60 Hz</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_Cargols">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>731</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Informe OF</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="taulaOF">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget_llancadora">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelLlarg">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string> Mida</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Crea OF</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCargols">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string> Tipus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
